rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/decompiler/FernflowerDecompilerOperation.java

```diff
@@ -326,6 +326,9 @@ private void handleDecompiledJavaFile(FileModel decompiledFileModel, FileService
                     // noop
                 }
             }
+
+            if (queueSize.get() > 1000)
+                LOG.warning("Decompiler queue size is: " + queueSize.get());
```

Comment:
what is this warning good for? when we have another for 100K above? I understand that first warning but the second is redundant or not?

Reply:
It shouldn't be there. I used that to monitor the queue size while testing other options than 5k. I will remove this line.